HerPortal.UnitTests/Website/Models/HomepageViewModelTests.cs

```diff
@@ -0,0 +1,42 @@
+﻿using FluentAssertions;
+using HerPortal.BusinessLogic.Models;
+using HerPortal.Models;
+using NUnit.Framework;
+
+namespace Tests.Website.Models;
+
+[TestFixture]
+public class HomepageViewModelTests
+{
+    [Test]
+    public void ShouldShowBannerIfUserHasNotLoggedIn()
```

Comment:
We're trying to go with a test naming pattern along the lines of `HomePageViewModel_WhenCreatedForUserThatHasntLoggedInBefore_ShouldShowBanner` i.e. `TheThingOrMethodBeingTested_TheCase_TheResult`